dashboard/app/views/pd/workshop_mailer/teacher_cancel_receipt.html.haml

```diff
@@ -0,0 +1,28 @@
+:css
+  body {
+    font-family: Gotham, sans-serif;
+  }
+
+%p
+  You have canceled your registration for the Code.org
+  = @workshop.course
+  = @workshop.subject
```

Comment:
Worth future-proofing with a function on @workshop that returns the full friendly name instead of joining in various places at render time?